### PR TITLE
fixed index.js lint errors

### PR DESCRIPTION
**Pull request by ElenaBorisenkoO**
please, check this:)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,201 +1,171 @@
...
-			if(arguments.length!==0 && (typeof arguments[0]==='function')){
-
-				for(let i=0; i<this.elements.length; i++){
-					
-					arguments[0](this.elements[i],i,this.elements);
-					resultArr[i] = this.elements[i];
-					
-				}
+MyArray.prototype.pop = function() {
+  // eslint-disable-next-line init-declarations
```
Comment (Jarosic) [inline src/index.js]: Why did you add such comments?

Comment (ElenaBorisenkoO) [reply src/index.js]: I'll fix it, thank you

**Review by Jarosic — APPROVED** [review]
Well done!

**Review by AndrZakhara — APPROVED** [review]
approved
